specification/marketplacecatalog/resource-manager/Products/preview/2022-09-01-preview/products.json

```diff
@@ -0,0 +1,1411 @@
+{
+  "x-generator": "NSwag v13.12.1.0 (NJsonSchema v10.4.6.0 (Newtonsoft.Json v13.0.0.0))",
```

Comment:
what does this line mean?

Reply:
This is just the swagger version that generated this openAPI docs json